src/components/ChatEntry.js

```diff
@@ -1,22 +1,62 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
+
+/**
+ *
+ * Component for showing details of individual messages
+ *
+ * @component
+ *
+ */
 
 const ChatEntry = (props) => {
+  const updateLikes = () => {
+    props.likedCallback(props.id);
+  };
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div
+      className={`chat-entry ${
+        props.sender === 'Estragon' ? 'remote' : 'local'
+      }`}
```

Comment:
Tip: This is a little brittle since if we dropped in a new JSON file (with different users) it would break.